test/src/snapshot.test.js

```diff
@@ -0,0 +1,26 @@
+const { BN } = require('../../src/setup');
+const { expect } = require('chai');
+
+const snapshot = require('../../src/snapshot');
+const time = require('../../src/time');
+
+describe('snapshot', function () {
```

Comment:
Can you add a test where you snapshot, then send some ether to another account, then restore the snapshot and confirm that the ether transfer was rolled back?